### DISCUSSION TITLE
Display background during osu!lazer launch animation

### DISCUSSION THREAD

**Discussion by Moltanicaa** [ppy/osu#15942] [Ideas]
If you enable screen scaling and set the scale to at least 99%x99% (any value, but not 100%), then during the welcome animation you will not have a black background, but a "base" lazer background.
![image](https://user-images.githubusercontent.com/59089206/144736261-2342656a-2615-4eae-a072-16b1acb19145.png)

Default (100% scaling/scaling off):
![osu_2021-12-05_09-24-32](https://user-images.githubusercontent.com/59089206/144736265-011c6829-e6ee-4d77-b276-be5b27853d25.jpg)

With 99% scaling:
![osu_2021-12-05_09-23-45](https://user-images.githubusercontent.com/59089206/144736274-c86a74eb-9e9e-4344-9207-ef5ea6149307.jpg)
And if you have a supporter, then after the disclaimer you will have a background of the map, which will play after the "welcome that osu":
![osu_2021-12-05_09-30-29](https://user-images.githubusercontent.com/59089206/144736323-5fe3d0ab-862b-415c-8f90-4f132b2cf43a.jpg)

So ... why not make it the default?

**Comment by peppy** [comment]
that looks unintentional

**Reply by bdach** [reply]
i think a decent solution would be to only fade in the "scaled" background in after the intro is done and the main menu is shown?

**Reply by peppy** [reply]
Potentially. Or just give the intro a black background (for the scaleable area).

**Reply by smoogipoo** [reply]
This almost looks like another issue we had and still haven't resolved, something like dim not being the same after screen scaling or something like that.

**Reply by peppy** [reply]
That one is due to masking being applied. Slightly different, only relationship is that they are both caused by having screen scaling enabled.

**Reply by Moltanicaa** [reply]
Is there a problem why this can't be turned into a good customization opportunity?

**Reply by peppy** [reply]
It's not intended. If you're looking for customisation please open a separate discussion. Or don't, because this kind of stuff is not going to be up to anyone but me to choose how it works, since I have a vision in mind when making the intro sequences and general menu logic.

**Reply by Moltanicaa** [reply]
So sad. Of course, I would like to raise this issue in the future. For example, to have a disclaimer with a black font on a white background or etc. This could be a good base for customization with a supporter.

**Reply by peppy** [reply]
The disclaimer is going to disappear soon. And no, font colours and stuff like that will not be customisable in menus. That is out of scope.

**Reply by Moltanicaa** [reply]
Got it. Thank you.
